src/selectors/per-thread/index.js

```diff
@@ -303,3 +303,107 @@ export const selectedNodeSelectors: NodeSelectors = (() => {
     getSourceViewLineTimings,
   };
 })();
+
+export const selectedFunctionTableNodeSelectors: NodeSelectors = (() => {
```

Comment:
Interesting. Did you add these selectors for performance reasons or as convenience helpers?

Reply:
The latter, but they also seem to offer better performance.